plugins/save/plugin.js

```diff
@@ -45,7 +45,7 @@
 				return;
 
 			var command = editor.addCommand( pluginName, saveCmd );
-			command.modes = { wysiwyg: !!( editor.element.$.form ) };
+			command.modes = { wysiwyg: !!( editor.element.$.form ), source: !!( editor.element.$.form ) };
```

Comment:
If at last we got to fix it, let's do it properly! I don't feel that `command.modes` is a proper place to do any of such checks (apparently [not only me](https://dev.ckeditor.com/ticket/8354#comment:1)). Modes should be just enabled and check should be moved e.g. to [`command.startDisabled`](https://docs.ckeditor.com/#!/api/CKEDITOR.commandDefinition-property-startDisabled).